packages/grafbase-sdk/tests/unit/federation.test.ts

```diff
@@ -2,7 +2,7 @@ import { config, graph } from '../../src/index'
 import { describe, expect, it, beforeEach } from '@jest/globals'
 import { renderGraphQL } from '../utils'
 
-const g = graph.Single()
+const g = graph.Standalone({ subgraph: true })
```

Comment:
I wonder if we should future-proof subgraph and make it an object that takes a few fields like enable, version, etc. Might be overthinking it.

Reply:
I don't think it'd be a breaking change to add those later